### PR TITLE
Adding doodles and placing them throughout.

### PR DESCRIPTION
**Pull request by davidrnovak**

### CODE REVIEW

File: src/_includes/deep-features.njk [hunk 1]
```diff
@@ -1,5 +1,6 @@
+<span class="deep-features-doodle"></span>
```
Comment (enatario) [inline src/_includes/deep-features.njk]: We can nix this

File: src/css/base/_button.scss [hunk 2]
```diff
@@ -22,3 +22,31 @@
...
+
+  &::before {
+    @include size(5rem);
+
+    background-image: url("/static/images/svg/red-swoop-arrow.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    content: "";
+    display: block;
+    // transform: translate(-5rem, 5rem);
```
Comment (enatario) [inline src/css/base/_button.scss]: Nix this line

File: src/css/base/_layout.scss [hunk 3]
```diff
@@ -85,3 +86,72 @@ footer {
...
+    display: none;
+
+    @media screen and (min-width: 30em) {
+      display: inline-flex;
+      height: 80rem;
+      transform: translateY(78rem) translateX(2rem);
+      width: 75rem;
+    }
+  }
+} */
```
Comment (enatario) [inline src/css/base/_layout.scss]: Nix

**Review by enatario — CHANGES_REQUESTED** [review]
Just a few lines to nix :)

**Review by davidrnovak — COMMENTED** [review]
Made all the requested changes. :)

**Review by enatario — APPROVED** [review]
ship iiiiit

**Pull request merged**